### ISSUE TITLE
Consider updating the visuals to better match the current GNOME visuals.

### ISSUE THREAD

**Issue by PauloVBettio** [oae/gnome-shell-pano#185]
The current visual does not match the GNOME visuals and ends up being too different from other applications that use Libadwaita.

I've made a concept showing a new design that is similar to the Libadwaita apps

![Pano](https://user-images.githubusercontent.com/74562432/233803012-59d7c1b7-911c-434c-bfd3-386796ede9ab.png)

**Comment by PauloVBettio** [comment]
Maybe an design that is like the shell Screenshot tool would make more sense too, and it looks very nice.

I don't know if it is possible to an extension to focus on a window like the Screenshot tool from the shell does, but that would be a nice feature too.

![PanoHorizontal](https://user-images.githubusercontent.com/74562432/235323374-295b23a0-1ce7-43de-9bb5-bfb4d75d2c2c.png)
![PanoVertical](https://user-images.githubusercontent.com/74562432/235323378-eda996b1-a1f8-425a-b718-476fca7a948e.png)

**Comment by 3laws** [comment]
I honestly was already sold on  the extension based on a text only description, later by coming here to GitHub to find out it didn't follow GNOME's HIG. Not a deal breaker but surely a very welcome addition. These mock ups look really good to be honest. Hopefully we can get something similar merged.

**Comment by g-lopes** [comment]
I love Pano, I use it every day and I like the way it looks now. With that said, this design looks awesome! It would be amazing if this became a reality.

**Comment by oae** [comment]
This looks really nice. I will experiment with Pano using these designs

**Comment by oae** [comment]
There is also another design by @callmehanu https://github.com/oae/gnome-shell-pano/discussions/170

**Comment by PauloVBettio** [comment]
> Yep... even I worked on a short prototype [Figma Prototype](https://www.figma.com/proto/hTFYs9qIwhfeDewIkKWshE/Pano-Realing?node-id=1%3A2&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=1%3A2) @PauloVBettio @g-lopes @3laws
> 
>     * Press "Q" to launch Panos
> 
>     * Press "I" to start search
> 
>     * Press "ESC" to quit search

The animations are sweet! And it fits the shell very well too. Awesome work!
I just think there should be a search button always visible, since its not everyone who will see the settings to see which one it is. And square is better than rectangular in this case, because more elements can fit in the screen.

**Comment by marcinjahn** [comment]
> I just think there should be a search button always visible, since its not everyone who will check the settings to see which one it is

Ideally, it should be configurable, IMO, searchbar should appear only as soon as you start typing.

**Comment by boerdereinar** [comment]
For anyone interested, I made my own [fork](https://github.com/boerdereinar/gnome-shell-pano) roughly following the concept.

![image](https://github.com/oae/gnome-shell-pano/assets/54003172/716ad6c0-b7eb-41a5-8a8b-705f8dc9819b)

**Comment by oae** [comment]
This looks good. It would be awesome to apply this design while keeping the customization features.

**Comment by Totto16** [comment]
> This looks good. It would be awesome to apply this design while keeping the customization features. What do you think @Totto16 ?

I like the proposed new design too, but the old (current) is also nice. To be fair they differ a lot, this looks more sleek, with more focus on the actual content and no "big" header.

"You can never have enough customization options" is also a good point, although we already have many options, it is also possible in my opinion, to add another option to have the new look, alltough that might not be worth the trouble and resulting "spaghetti code" 😂


What speaks against immediately adopting this is, that all tough the current style is older, it's also better in terms of functionality IMO, since you can immediately see the big header, and immediately see the type.

The round styles and headerless design is really good per sè, but we would need to improve a few points and merge the good suggestion in this issue to get a good version.

All in all I find the new one good, but the old one too, and I am torn apart, which is better imo, but in the long run the newer, sleeker one is better.

TLDR: 🤷🏼‍♂️  😓

**Comment by marcinjahn** [comment]
I can only imagine how terrible it must be to introduce any new features at the moment, with the amount of customization that the extension allows, and potential places where things can break. I think it’d be best to switch to some adwaita style, and remove appearance customization options, all leave only some basic ones. Maintenance of the project should become much easier then

**Comment by Totto16** [comment]
Customizations aren't terrible to maintain and don't break easy, e.g. setting the color means that we retrieve the color from a setting instead of hard-coding it, that's't not difficult and error prone.

But the entire style, like header, widgets etc. is a little more complicated, thats error-prone and harder to maintain for future gnome versions

**Comment by boerdereinar** [comment]
Made a second version:
- All customisation options that are still relevant, work
- Added `Compact Mode` that is useful in vertical mode
- Added option to only show controls on hover
- Files item now shows the number of files that were copied/cut
- Files item now renders as a tree branching from the top level directory. This makes it so that individual file names can actually be read
- Color item now follows the prototype design and picks the font color automatically so that it should stay legible for any color

https://github.com/boerdereinar/gnome-shell-pano/tree/gnome-theming-v2

Controls only on hover:
![image](https://github.com/oae/gnome-shell-pano/assets/54003172/cdedc42d-f91a-4b1c-a0ea-47fffaa7a40a)

Compact Mode:
![image](https://github.com/oae/gnome-shell-pano/assets/54003172/a9706bb5-6de5-4cde-9706-c3b7e92bc2fd)

Customisation:
![image](https://github.com/oae/gnome-shell-pano/assets/54003172/b0481f49-d443-4283-86b3-c3236fc5a3b3)

**Comment by boerdereinar** [comment]
Headers can now be enabled again.

![image](https://github.com/oae/gnome-shell-pano/assets/54003172/9e0efde0-9d6c-410e-8c59-bf63a780cc2b)
